### PR TITLE
Create LICENSE.md

### PR DESCRIPTION
**Pull request by muneebahmed10**
Adds MIT license, copyright holder Amazon Web Services

### CODE REVIEW

File: LICENSE.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+MIT License
+
```
Comment (aggarw13) [inline LICENSE.md:2]: Is this the same as the C-SDK repository?

Comment (muneebahmed10) [reply LICENSE.md:2]: Yes, it's identical

**Review by aggarw13 — APPROVED** [review]

**Comment by sarenameas** [comment]
Looks like we can close this?

**Comment by muneebahmed10** [comment]
Looks like this file was added in another commit without a PR. Unsure if we still need to add a copyright holder but that would be a separate PR

**Pull request closed without merging**